src/util/antlr/ANTLRErrorHandling.h

```diff
@@ -4,21 +4,54 @@
 
 #pragma once
 
+#include <parser/ParseException.h>
+
 #include <string>
 
-#include "../../parser/ParseException.h"
 #include "BaseErrorListener.h"
+#include "CharStream.h"
+#include "CommonTokenStream.h"
 #include "DefaultErrorStrategy.h"
+#include "ParserRuleContext.h"
 #include "RecognitionException.h"
+#include "Recognizer.h"
+#include "TokenSource.h"
+
+// TODO<qup42> whats up with this namespace?
+namespace {
```

Comment:
In headers you make such functions `inline` (they shall be the same in all translation units).
anonymous namespace is a header as not a real pattern.
But what you maybe actually should do here is extract the definitions into a `.cpp`-File and add comments in the header.